### PR TITLE
Move FAQ from Toolkit

### PR DESCRIPTION
**Pull request by Yoshanuikabundi**
This PR re-organizes and replaces the Toolkit FAQ to be part of the top level docs.

Rendered: https://openff-docs--63.org.readthedocs.build/en/63/faq.html

Todo:
- [x] Add corrections from openforcefield/openff-toolkit#1848
- [x] Add section about warnings re: openforcefield/openff-toolkit#1932

### CODE REVIEW

**Comment by Yoshanuikabundi** [comment]
@j-wags I've made some edits to what Matt brought up in openforcefield/openff-toolkit#1848, so I'm going to leave this unmerged for tonight in case he has corrections - I should be able to merge it in my AM before your talk.

@mattwthompson - sorry for dropping off the face of the planet and then asking for a quick response :/ I'm happy to merge this as is and you can get around to checking it at your leisure - I'm confident its correct, just not sure it addresses exactly the points you wanted to address.

**Comment by Yoshanuikabundi** [comment]
Do we have a way to remove parameters from a force field? We've got `del` in the FAQ:

```python
del force_field['Constraints']["[#1:1]-[*:2]"]
```

But it doesn't work. Does anyone know if this is a change or did this just never work?

**Comment by mattwthompson** [comment]
I can get below to work with some effort, which I can't say would make me super happy as a user:


```python
In [13]: sage = ForceField("openff-2.0.0.offxml")

In [14]: sage['Constraints']["[#1:1]-[*:2]"]
Out[14]: <ConstraintType with smirks: [#1:1]-[*:2]  id: c1  >

In [15]: sage["Constraints"].parameters.index(sage['Constraints']["[#1:1]-[*:2]"])
Out[15]: 0  # not relevant, just checking

In [16]: del sage['Constraints'].parameters["[#1:1]-[*:2]"]

In [17]: sage['Constraints']["[#1:1]-[*:2]"]
---------------------------------------------------------------------------
ParameterLookupError                      Traceback (most recent call last)
Cell In[17], line 1
----> 1 sage['Constraints']["[#1:1]-[*:2]"]

File ~/micromamba/envs/evaluator-test-env/lib/python3.11/site-packages/openff/toolkit/typing/engines/smirnoff/parameters.py:2439, in ParameterHandler.__getitem__(self, val)
   2434 def __getitem__(self, val):
   2435     """
   2436     Syntax sugar for looking up a ParameterType in a ParameterHandler
   2437     based on its SMIRKS.
   2438     """
-> 2439     return self.parameters[val]

File ~/micromamba/envs/evaluator-test-env/lib/python3.11/site-packages/openff/toolkit/typing/engines/smirnoff/parameters.py:1574, in ParameterList.__getitem__(self, item)
   1572     indexable_item: Union[int, slice] = item  # type: ignore[assignment]
   1573 elif isinstance(item, str):
-> 1574     indexable_item = self.index(item)
   1575 elif isinstance(item, ParameterType) or issubclass(type(item), ParameterType):
   1576     raise ParameterLookupError("Lookup by instance is not supported")

File ~/micromamba/envs/evaluator-test-env/lib/python3.11/site-packages/openff/toolkit/typing/engines/smirnoff/parameters.py:1529, in ParameterList.index(self, item, start, stop)
   1527     if parameter.smirks == item:
   1528         return self.index(parameter)
-> 1529 raise ParameterLookupError(f"SMIRKS {item} not found in ParameterList")

ParameterLookupError: SMIRKS [#1:1]-[*:2] not found in ParameterList

```

I think https://github.com/openforcefield/openff-toolkit/issues/1680 is related if we want to directly add this functionality to pass through `ParameterHandler`s - notably `class ParameterList(list):`, but other handy features allow users to bypass explicitly calling out `.parameters`

File: source/index.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ Specifications for SMIRNOFF and other standards defined by OpenFF can be found a
 : Submit and retrieve datasets with rich metadata from QCFractal instances
 
 [`openff-fragmenter`](https://docs.openforcefield.org/projects/fragmenter)
-: Fragment molecules for efficient quantum mechanical torsion scans 
+: Fragment molecules for efficient quantum mechanical torsion scans
```
Comment (mattwthompson) [inline source/index.md:34]: https://github.com/pre-commit/pre-commit-hooks?tab=readme-ov-file#trailing-whitespace would standardize against what our different IDEs (and developers) sneak or accidentally let in

File: source/faq.md [hunk 2]
```diff
@@ -0,0 +1,308 @@
...
+
+:::::
+
+:::::{faq-entry} Can I use an Amber or GROMACS topology/coordinate file as a starting point for applying a SMIRNOFF force field?
+
+Amber and GROMACS topologies and coordinate files do not include enough explicit chemical information to apply a SMIRNOFF force field.
+For example, bond orders are not present in either format; one could infer bond orders based on bond lengths, or attempt to infer bond orders from force constants, but such inference work would be error-prone and is outside the scope of SMIRNOFF.
+PDB files that include all atoms in the model can be used in some cases (see next question).
+
+Amber and GROMACS topology and coordinate files can be [experimentally loaded] by Interchange for export to other MD engines, but this does not require the chemical information needed to apply a SMIRNOFF force field.
```
Comment (mattwthompson) [inline source/faq.md:39]:
```suggestion
Amber and GROMACS topology and coordinate files can be [experimentally loaded] by Interchange for export to other MD engines, but this does not include the chemical information needed to apply a SMIRNOFF force field.
```

File: source/faq.md [hunk 3]
```diff
@@ -0,0 +1,308 @@
...
+
+## Errors and Performance Issues
+
+:::::{faq-entry} Why does partial charge assignment fail during conformer generation, even though my molecule has conformers?
+
+Assigning partial charges with a quantum chemical method requires conformers, as they are an essential input to a quantum chemical calculation.
+Because the charges assigned by a SMIRNOFF force field should be transferrable between systems, we default to generating our own set of conformers during charge assignment.
+This requirement will become unnecessary for future SMIRNOFF force fields that use NAGL graph charges; see the [](#under-the-hood) section.
+
+To assign charges based on the provided conformer if conformer generation fails, first assign charges, then use the assigned charges during parametrization:
```
Comment (mattwthompson) [inline source/faq.md:112]: I get the message here, but the phrasing is (to me) not detailed enough on the specific point of "you can use the API to use _your_ conformer(s) if conformer generation fails." The `use_conformers` argument isn't explicitly called out in the sentences above and I think it'd be useful to call it out here, i.e.
```suggestion
To assign charges based on the provided conformer if conformer generation fails, you can use a `try:`/`except:` block and the `use_conformers` argument when assigning charges. These charges can then be used charges during parametrization with the `charge_from_molecules` argument:
```

**Review by mattwthompson — APPROVED** [review]
LGTM; I sought out only the section headers that seemed relevant to the questions I asked back in march

**Pull request merged**